### ISSUE TITLE
User with username only is not recognized in latest version 1.2.6

### ISSUE THREAD

**Issue by lukvac** [chadwpalm/Lumunarr#24]
My example local user called just "TV" stopped worked from this version.

**Comment by chadwpalm** [comment]
Yeah, the problem is a mismatch with webhook using the user account "title" and not the "username" and managed accounts don't have a username. I tried to fix one problem in 1.2.6 and ended up creating another.

Ultimately I should just be using the user ID to match on the backend. I store the user ID in the config file.....not sure why I chose the username to match during the hook when I should be matching the ID and avoid the title/username mismatches altogether.

I'm probably also going to revert the change I made to display the title instead of the username on the client cards in the UI...that way when you set up a managed user it won't be blank.

Anyway, l will fix the issue and send out a hotfix to the dev branch sometime tonight (US West Coast time). I will probably be 1.3.1.

Chad
